src/schema.ts

```diff
@@ -19,7 +18,6 @@ export function createSchema<TApi, TData>(services: Array<Service<TApi, TData>>)
   const rootSchema = makeExecutableSchema({
     typeDefs: `
       scalar JSON
-      scalar Upload

```

Comment:
Is this still tbd or removed on purpose?

Reply:
ugh, removed on purpose

Reply:
Reason? We still want the `Upload` field to be available.

Reply:
well, tbh it should not be removed since the scheme is being created with `makeExecutableSchema` Just noticed a clarification in [docs](https://github.com/apollographql/apollo-server/blob/main/docs/source/data/file-uploads.md).
> Note: When using typeDefs, Apollo Server adds scalar Upload to your schema, so any existing declaration of scalar Upload in the type definitions should be removed. If you create your schema with makeExecutableSchema and pass it to ApolloServer constructor using the schema param, make sure to include scalar Upload.